clients/redis.go

```diff
@@ -0,0 +1,25 @@
+package clients
+
+import (
+	"context"
+	"fmt"
+	"github.com/Vertamedia/chproxy/config"
+	"github.com/go-redis/redis/v8"
+)
+
+func NewRedisClient(cfg config.RedisCacheConfig) (redis.UniversalClient, error) {
```

Comment:
Improvement for the future: Implement TLS Client

Reply:
I'll add a todo